kuma/search/tests/test_views.py

```diff
@@ -1,32 +1,28 @@
 # -*- coding: utf-8 -*-
 import elasticsearch
+import pytest
 
-from kuma.core.tests import eq_
 from . import ElasticTestCase
+from ..models import Filter, FilterGroup, Index
 from ..pagination import SearchPagination
-from ..models import Index, Filter, FilterGroup
 from ..views import SearchView
 
 
 class ViewTests(ElasticTestCase):
     fixtures = ElasticTestCase.fixtures + ['wiki/documents.json',
                                            'search/filters.json']
 
-    def test_search_rendering(self):
-        """The search view """
-        response = self.client.get('/en-US/search?q=test')
-        eq_(response.status_code, 200)
-        self.assertContains(response, 'Results for')
-
-        response = self.client.get('/en-US/search?q=article')
-        eq_(response.status_code, 200)
-        self.assertContains(response, 'an article title')
-
     def test_search_filters(self):
+        """The search view """
```

Comment:
👍 combining two tests that seem to do the same thing
👎 keeping the same silly docstring. It should be removed or be upgraded to be more descriptive.

Reply:
Oops, good catch!